src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -45,3 +45,5 @@ Our operators will create this `Grafana` organization named _Giant Swarm_. It wi
 
```

Comment:
In the line above there is a mistake I think. The example Organisation CRD shows "myonlineshop" as a listed tenant, but the text beneath says it gives you access to the giantswarm tenant - we either have to refine that to "allowing you access to the `myonlineshop` tenant" or change the tenant in the yaml, but then I'd add "gives you access to the shared giantswarm tenant"